modules/rest_api/controllers/services/rest.php

```diff
@@ -1044,16 +1044,40 @@ private function checkAllowedResource($proj_id, $resourceName) {
       ['caption' => 'Comment', 'field' => 'occurrence.occurrence_remarks'],
       ['caption' => 'Sample comment', 'field' => 'event.event_remarks'],
       ['caption' => 'Images', 'field' => '#occurrence_media#'],
-      ['caption' => 'Input on date', 'field' => 'metadata.created_on'], // Format ??
-      ['caption' => 'Last edited on date', 'field' => 'metadata.updated_on'], // Format ??
-      ['caption' => 'Verification status 1', 'field' => 'identification.verification_status'],
-      ['caption' => 'Verification status 2', 'field' => '#null_if_zero:identification.verification_substatus#'],
-      ['caption' => 'Query', 'field' => 'identification.query'],
+      ['caption' => 'Input on date', 'field' => '#datetime:metadata.created_on:d/m/Y H\:i#'],
+      ['caption' => 'Last edited on date', 'field' => '#datetime:metadata.updated_on:d/m/Y H\:i#'],
+      ['caption' => 'Verification status 1', 'field' => '#verification_status:standard#'],
+      ['caption' => 'Verification status 2', 'field' => '#verification_substatus:standard#'],
+      ['caption' => 'Query', 'field' => '#query:standard#'],
       ['caption' => 'Verifier', 'field' => 'identification.verifier.name'],
-      ['caption' => 'Verified on', 'field' => 'identification.verified_on'],
+      ['caption' => 'Verified on', 'field' => '#datetime:identification.verified_on:d/m/Y H\:i#'],
+      ['caption' => 'Licence', 'field' => 'metadata.licence_code'],
+      ['caption' => 'Automated checks', 'field' => 'identification.auto_checks.result'], 
+    ],
+    "mapmate" => [
+      ['caption' => 'Taxon', 'field' => 'taxon.accepted_name'],
+      ['caption' => 'Site', 'field' => 'location.verbatim_locality'],
+      ['caption' => 'Gridref', 'field' => 'location.output_sref'],
+      ['caption' => 'VC', 'field' => '#vc:mapmate#'],
+      ['caption' => 'Recorder', 'field' => 'event.recorded_by'],
+      ['caption' => 'Determiner', 'field' => 'identification.identified_by'],
+      ['caption' => 'Date', 'field' => '#record_date:mapmate#'],
+      ['caption' => 'Quantity', 'field' => '#organism_quantity:mapmate#'],
+      ['caption' => 'Method', 'field' => 'event.sampling_protocol'],
+      ['caption' => 'Sex', 'field' => '#sex:mapmate#'],
+      ['caption' => 'Stage', 'field' => '#life_stage:mapmate#'],
+      ['caption' => 'Status', 'field' => '#blank#'],
+      ['caption' => 'Comment', 'field' => '#sample_occurrence_comment#'],
+      ['caption' => 'ID', 'field' => 'id'],
+      ['caption' => 'RecordKey', 'field' => '_id'],
+      ['caption' => 'NonNumericQuantity', 'field' => '#organism_quantity:non-integer#'],
```

Comment:
Is exclude_integer a better name for the format specifier?

Reply:
Agreed.

Reply:
Complete.